ain_imager_src/changeproperty.cpp

```diff
@@ -137,6 +137,15 @@ void ImagerWindow::change_agent_gain_property(const char *agent, QSpinBox *ccd_g
 	indigo_change_number_property_1(nullptr, agent, CCD_GAIN_PROPERTY_NAME, CCD_GAIN_ITEM_NAME, value);
 }
 
+void ImagerWindow::change_agent_temperature_compensation_steps(const char *agent) const {
+	static const char *items[] = {
+		FOCUSER_COMPENSATION_ITEM_NAME
+		};
+	static double values[1];
+	values[0] = (double)m_focuser_temperature_compensation_steps->value();
+	indigo_change_number_property(nullptr, agent, FOCUSER_COMPENSATION_PROPERTY_NAME, 1, items, values);
```

Comment:
better use indigo_change_number_property_1() finction

Reply:
OK. Changed to _1